src/rustup-cli/self_update.rs

```diff
@@ -1069,7 +1071,7 @@ pub fn update() -> Result<()> {
     }
     let setup_path = try!(prepare_update());
     if let Some(ref p) = setup_path {
-        info!("rustup updated successfully");
+        info!("rustup updated successfully to {}", VERSION);
```

Comment:
By printing `CARGO_PKG_VERSION` here rustup will be printing the version of *the currently running version of rustup*. Instead, what we want is to print the version *we just downloaded* (here represented by `setup_path`). So this is instead going to need to run the executable at `setup_path` with `--version` and parse out the new version number.